src/txn/txn_rollback_to_stable.c

```diff
@@ -266,14 +240,15 @@ __rollback_check_if_txnid_non_committed(WT_SESSION_IMPL *session, uint64_t txnid
 }
 
 /*
- * __rollback_row_ondisk_fixup_key --
+ * __rollback_ondisk_fixup_key --
  *     Abort updates in the history store and replace the on-disk value with an update that
  *     satisfies the given timestamp.
  */
 static int
-__rollback_row_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW *rip,
-  wt_timestamp_t rollback_timestamp, bool replace)
+__rollback_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_REF *ref, WT_PAGE *page, WT_COL *cip,
```

Comment:
`page` can be accessed from `ref` and it is the same for both row or column store. You can pass only `ref`.

Reply:
I changed the row store callers to just use REF now but __rollback_abort_row_reconciled_page_internal, creates a new page that it passes to this function so I don't think I can remove it from the arguments.

Reply:
WT-5949 is to remove the `reconcile_page` function completely. As part of that ticket, we can do this cleanup to simplify the code logic.

Reply:
Okay, I'll leave it for now then.